python/delta-kernel-rust-sharing-wrapper/src/lib.rs

```diff
@@ -4,17 +4,19 @@ use arrow::compute::filter_record_batch;
 use arrow::datatypes::SchemaRef;
```

Comment:
Note: Both arrow and kernel have their own `SchemaRef`. This is up to personal preference, but you may want to give this an alias so that the type of `SchemaRef` doesn't become confusing.
```suggestion
use arrow::datatypes::SchemaRef as ArrowSchemaRef;
```

Reply:
You may also want to do the same for `TableChangesScan`:
```rust
use delta_kernel::table_changes::scan::TableChangesScan as KernelTableChangesScan
```

Reply:
I think `ArrowSchemaRef` makes sense, but `TableChangesScan` is only used once so I don't think using an alias for it is as beneficial.